tools/tox.ini

```diff
@@ -4,7 +4,10 @@ skipsdist=True
 
 [testenv]
 deps =
-  flake8
+  flake8==3.5.0
```

Comment:
You might want to point out that these version numbers are duplicated elsewhere in the repo.